### ISSUE TITLE
Create a User Registration page with form validation

### ISSUE THREAD

**Issue by adarsh-technocrat** [codeforcauseorg/edu-client#313]
**Suggested UI for the user registration page**

![Registration page](https://user-images.githubusercontent.com/47661086/117549177-7ca81b80-b056-11eb-852c-aec7982e5a22.png)

Cc/ @Abhishek-kumar09  @vasudevsinghal

**Comment by adarsh-technocrat** [comment]
Hey, @Abhishek-kumar09  you can assign this issue to me!

**Comment by Abhijay007** [comment]
> Hey, @Abhishek-kumar09 can you assign this issue to me!

@adarsh-technocrat I think this can be covered authentication part through firebase authentication service which includes both sign-up and sign-in option.

**Comment by adarsh-technocrat** [comment]
> @adarsh-technocrat I think this can be covered authentication part through firebase authentication service which includes both sign-up and sign-in option.

Yaa! @Abhijay007 actually project doesn't have that sign-in and sign-up page connected to the app right now so I thought of making this route and as those sections are up we can add this to it and further we can add phone Authentication as well :)

**Comment by Abhishek-kumar09** [comment]
Bolds are looking wierd and textFields too, something that better fits with the theme will be better.

**Comment by adarsh-technocrat** [comment]
> Bolds are looking weird and text fields too, something that better fits with the theme will be better.

Right! @Abhishek-kumar09 I have revised the design I'll share that one :)

**Comment by adarsh-technocrat** [comment]
@Abhishek-kumar09  ping!
